### PR TITLE
NR-83112 : provision canaries

### PR DESCRIPTION
**Pull request by brushknight**

### CODE REVIEW

File: test/canaries/roles/otel-dockerized/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  file:
+    path: "/srv/otel_collector/"
+    state: directory
+    mode: '0755'
+- name: copy compose to the machine
+  template:
+    src: ../files/docker-compose.yml.j2
+    dest: "/srv/otel_collector/otel_collector.yml"
+    mode: 0644
+- name: set previous display_name
```
Comment (rogercoll) [inline test/canaries/roles/otel-dockerized/tasks/main.yml:12]: What do you think setting `display_name` in the docker-compose template?
Same Ansible variables can be used and we decouple env vars from the role.

File: test/provision/terraform/install-requirements.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 
-- hosts: testing_hosts_linux
+- hosts: all
```
Comment (rogercoll) [inline test/provision/terraform/install-requirements.yml:3]: ❤️

**Review by rogercoll — COMMENTED** [review]
Nice! Left a few nits to discuss

File: test/canaries/roles/otel-dockerized/tasks/main.yml (hunk 1 above)
Comment (brushknight) [reply test/canaries/roles/otel-dockerized/tasks/main.yml:12]: Do you have an idea how to set it? 🤔

**Review by rogercoll — APPROVED** [review]
LGTM

**Pull request merged**